pkgs/development/libraries/snappy/default.nix

```diff
@@ -17,7 +17,10 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  cmakeFlags = [ "-DBUILD_SHARED_LIBS=ON" "-DCMAKE_SKIP_BUILD_RPATH=OFF" ];
+  cmakeFlags = [
+    "-DCMAKE_SKIP_BUILD_RPATH=OFF"
+    "-DBUILD_SHARED_LIBS=${if static then "OFF" else "ON"}"
```

Comment:
I wonder if not changing the order of the elements would reduce the number of rebuilds.

Reply:
Ah, this is exclusive shared or static, I guess.